recOrder/tests/cli_tests/test_cli.py

```diff
@@ -1,10 +1,154 @@
 from recOrder.cli.main import cli
+import unittest
 from click.testing import CliRunner
-
+from unittest.mock import patch, Mock
+from iohub.ngff import open_ome_zarr, Position
+from hypothesis import given, HealthCheck, settings
+from hypothesis import strategies as st
+from recOrder.cli.compute_transfer_function import (
+    generate_and_save_phase_transfer_function, 
+    generate_and_save_birefringence_transfer_function,
+    compute_transfer_function_cli
+)
+from recOrder.cli.settings import (
+    TransferFunctionSettings,
+    _PhaseTransferFunctionSettings,
+    _BirefringenceTransferFunctionSettings,
+    _UniversalSettings
+)
+import pytest
 
 def test_main():
     runner = CliRunner()
     result = runner.invoke(cli)
 
     assert result.exit_code == 0
     assert "Toolkit" in result.output
+
+def test_compute_transfer(tmp_path):
+    runner = CliRunner()
+    result = runner.invoke(cli, "compute-transfer-function -o " + str(tmp_path))
+    assert result.exit_code == 0
+    assert "Generating" in result.output
+
+def test_compute_transfer_blank_config():
+    runner = CliRunner()
+    for option in ("-c ", "--config-path "):
+        cmd = "compute-transfer-function " + option
+        result = runner.invoke(cli, cmd)
+        assert result.exit_code == 2
+        assert "Error" in result.output
```

Comment:
Here you are testing that if no config is provided it will raise an error. Yet in this line:
https://github.com/mehta-lab/recOrder/blob/6f8f05f500c9298053caeeba6f35788b738317e4/recOrder/cli/compute_transfer_function.py#L28
it is actually allowed. What is the indended behavior when no config is given?

Reply:
I believe that is for if they add the options "-c" or "--config-path" but do not provide a specific path for the argument. I assumed that the `config_path` being `None` was for having no option and argument, but I didn't realize that it may be for no given config. If we want the command to take in `recorder compute-transfer-function -c ` and still do the default configurations, I can look to make those changes.